contracts/eosio.system/src/powerup.cpp

```diff
@@ -278,8 +279,6 @@ int64_t calc_rentbw_fee(const rentbw_state_resource& state, int64_t utilization_
                coefficient * std::pow(end_u, state.exponent) - coefficient * std::pow(start_u, state.exponent);
    };
 
-   // Returns p(double(utilization)/state.weight).
```

Comment:
Why remove?

Reply:
Added the comment back. That was removed in error.